components/org.wso2.carbon.identity.api.server.api.resource/org.wso2.carbon.identity.api.server.api.resource.v1/src/main/resources/APIResources.yaml

```diff
@@ -0,0 +1,831 @@
+openapi: 3.0.0
+info:
+  description: >
+    This document specifies an **API Resource Management RESTful API** for **WSO2
+    Identity Server**.
+  version: "v1"
+  title: WSO2 Identity Server - API Resource Management Rest API
+  termsOfService: 'http://swagger.io/terms/'
+  contact:
+    name: WSO2
+    url: 'http://wso2.com/products/identity-server/'
+    email: architecture@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+
+security:
+  - OAuth2: []
+  - BasicAuth: []
+
+servers:
+  # Added by API Auto Mocking Plugin
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/THAMINDUA_1/api_resource/v1
+  - url: 'https://{server-url}/t/{tenant-domain}/api/server/v1'
+    variables:
+      tenant-domain:
+        default: "carbon.super"
+      server-url:
+        default: "localhost:9443"
+
+paths:
+  /api-resources:
+    post:
+      tags:
+        - API Resources
+      operationId: addAPIResource
+      summary: Add a new API resource
+      description: Add a new API resource
```

Comment:
Add the required permission and scope in the descripton for each endpoints

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/473/commits/5c51c9eff45d1c55a16c7858a056b5a9eed2ce86